examples/toolbox-demo/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "blockly-toolbox-demo",
+  "version": "0.0.0",
+  "private": true,
+  "demoConfig": {
```

Comment:
I'd like to have "see also" links for a lot of these ported demos. Do you think that belongs in the demoConfig, or should just go in the readme?